pages/instructorQuestionSettings/instructorQuestionSettings.ejs

```diff
@@ -84,6 +84,30 @@
               <th>Assessments</th>
               <td><% if (locals.assessments) { %><%- include('../partials/assessments', {assessments: assessments}); %><% } %></td>
             </tr>
+            <tr>
+              <th>Sharing Sets</th>
+              <td>
+                <% sharing_sets_in.forEach(function(sharing_set) { %>
+                <%= sharing_set.name %>,
+                <% }); %>
+                <form name="sharing-set-add-<%= question.id %>" method="POST" class="d-inline">
+                  <input type="hidden" name="__action" value="sharing_set_add">
+                  <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                  <div class="btn-group btn-group-sm" role="group">
+                    <button id="addSharingSet-<%= question.id %>" type="button" class="btn btn-sm btn-outline-dark dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
As above, it shouldn't be necessary to include the question ID in this ID.

Reply:
removed